assets/skins/CharacterScreen.skin

```diff
@@ -6,7 +6,7 @@
             "background" : "greyedBackground"
         },
         "UILabel" : {
-            "font" : "title"
+            "font" : "NotoSans-Regular-Large"
```

Comment:
You'll want to open the character screen with 'C', I opened it and it's got a lot of overlap!